rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1661,7 +1661,7 @@ def test_latest_upgrade_adds_remove_tables(user_data_dir):
     missing_views = views_before - views_after_upgrade
     assert missing_tables == removed_tables
     assert missing_views == removed_views
-    assert tables_after_creation - tables_after_upgrade == set()
+    assert tables_after_creation - tables_after_upgrade == {'optimism_transactions'}
```

Comment:
this should not be like this. Which is why I mentioned to add the addition at the upgrade